src/google/protobuf/port_def.inc

```diff
@@ -888,11 +888,18 @@ static_assert(PROTOBUF_ABSL_MIN(20230125, 3),
 #undef PACKAGE
 #endif
 
-// autoheader defines this in some circumstances
-#ifdef PACKAGE
-#define PROTOBUF_DID_UNDEF_PACKAGE
-#pragma push_macro("PACKAGE")
-#undef PACKAGE
+// a few common headers define this
+#ifdef PACKED
+#define PROTOBUF_DID_UNDEF_PACKED
+#pragma push_macro("PACKED")
+#undef PACKED
```

Comment:
It's worth noting that the section for dealing with `PACKAGE` is duplicated and that duplication got removed in #12903.

`PACKED` got added by https://github.com/protocolbuffers/protobuf/commit/7dbe742e6c4bea916c54713e31811412d2076351, which is effectively backported here through the conflict resolution.

Reply:
Thanks